tests/test_runner/test_parallel.py

```diff
@@ -72,29 +76,66 @@ def test_was_successful_no_events(self):
 
     def test_was_successful_one_success(self):
         result = RemoteTestResult()
-        result.addSuccess(None)
+        test = None
+        result.startTest(test)
+        result.addSuccess(test)
+        result.stopTest(test)
```

Comment:
Why do we have these test changes? When I revert them, the tests still pass

Reply:
It's true that these 5 tests still pass if `startTest()` and `stopTest()` are not called. I don't feel strongly about these changes.

The reason I added these is to make the tests more realistic. A side effect of `startTest()` is that it changes `RemoteTestResult.test_index` from `-1` to `0`. For these 5 tests, the tested `RemoteTestResult` method would only be called when `test_index` is >= `0` in practice.

(More about being realistic: `unittest.TestCase.run()` always calls `startTest()` before running a test and always calls `stopTest()` after. `startTest()` increments the `testsRun` attribute (`self.testsRun += 1`), which is used by the `RemoteTestResult.test_index` property. `startTest()` also has other side effects that `stopTest()` restores.)

(Come to think of it, if we keep these, then `stopTest()` should probably be called inside a `finally:`. I'll do that soon in case we keep these.)

Of these 5 tests, keeping the `startTest()` and `stopTest()` calls makes the most sense for `test_was_successful_one_error`. Otherwise it's very similar to `test_add_error_before_first_test`. (Calling `startTest()` in `test_was_successful_one_error` was actually necessary on this branch until this commit was added: "Reduced risk of breaking RemoteTestResult dependents.")